cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -562,3 +562,16 @@ def _cleanup():
     src_vm = form_data_vm_obj_single_datastore.vm_list[0]
     migrated_vm = get_migrated_vm_obj(src_vm, v2v_providers.rhv_provider)
     assert src_vm.mac_address == migrated_vm.mac_address
+
+
+@pytest.mark.parametrize('form_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal],],
+                        indirect=True)
+def test_edit_mapping_description(request, appliance, host_creds,
```

Comment:
Looks good. Maybe we can add few more tests around different edits and validate this by checking the Infra Mapping details in the `InfraMappingsList`

Reply:
Kedar will add that in next stage . For now I wanted to check if the framework to edit mapping works , so starting with just editing description